src/invite/InviteModal.tsx

```diff
@@ -77,10 +111,28 @@ const styles = StyleSheet.create({
     width: 120,
     height: 120,
   },
+  helpContainer: {
+    width: 312,
+    height: 100,
+    left: 24,
```

Comment:
i don't think these styles are quite working. i'd recommend not setting the width and height and position, and letting the container size itself based on the content (otherwise you risk clipped text and weird alignment depending on the screen size and device font scaling). all that's really needed is a marginBottom i think, and for spaces we should use the `Spacing` variable.

![Simulator Screen Shot - iPhone SE (3rd generation) - 2023-03-17 at 11 36 03](https://user-images.githubusercontent.com/20150449/225881962-57d5fe18-f15d-415b-bdbc-26c1118a6731.png)

Reply:
oh okay sounds good. I copied from figma, but thought it didn't look right on different devices, and didn't know what to do, will add the margin and remove the absolute positioning